docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+.. _kinto-glossary:
+
+Glossary
+########
+
+
+.. glossary::
+
+    Cliquet Protocol
+            A system of rules that explains the way to interact with Kinto,
```

Comment:
I thought it was more than that. For instance "GET /" is part of the protocol, as defined in Cliquet docs, and it's not a Resource (according to the definition in the Cliquet glossary)